CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Documentation for rocSPARSE is available at
 ### Removed
 
 * The deprecated `rocsparse_spmv_ex` routine
+* The deprecated `rocsparse_sbsrmv_ex`, `rocsparse_dbsrmv_ex`, `rocsparse_cbsrmv_ex` and `rocsparse_zbsrmv_ex`  routines
+* The deprecated `rocsparse_sbsrmv_ex_analysis`, `rocsparse_dbsrmv_ex_analysis`, `rocsparse_cbsrmv_ex_analysis` and `rocsparse_zbsrmv_ex_analysis`  routines
 
```

Comment:
Maybe also add that we added additional warnings when falling back to other algorithms.

Reply:
These warnings are only available when calling `rocsparse_debug_enable`, or exporting the environment variable ROCSPARSE_DEBUG with an assigned value of 1. Do you still want me to add the note?

Reply:
I added the note.